doc/software-design-development-guidance/index.md

```diff
@@ -0,0 +1,267 @@
+
+SAP (Systems, Applications and Processing) is an enterprise software applications provider that helps businesses of all sizes run better. The company’s software acts as a centralized system that allows departments to access and share data, and at the heart of these different software is the SAP Fiori for Web design system. 
+
+Built with the company’s SAPUI5 framework, which is a collection of user interfaces for web apps, the company places an emphasis on designing for common components, including navigational elements, tables and user inputs. Big picture, the brand includes guidelines for general principles, using layouts, patterns and controls, as well as concrete guidance to execute a user-centered approach to designing SAP Fiori apps.
+
+### Component framework: shadcn
+
+[shadch: build your component library](https://ui.shadcn.com/)
+
+TODO
+
+### Component framework: Flowbite
+
+[Flowbite: Build websites even faster with components])https://flowbite.com/)
+
+TODO
+
+### Component framework: SkeletonUI
+
+[SkeletonUI adaptive design system](https://www.skeleton.dev/)
+
+TODO
+
+## Recommendation - Decision Outcome
+
+Given what we know so far, and the immediacy of the emergency department module, we are immediately pursuing the combination of one choice in each category that we believe will keep the most options open for our ongoing improvements and learning. In other words, we need to kickstart real work, and we know that what we learn about our needs may well alter our longer-term choices.
+
+* Government guidelines: We are kickstarting with GOV.UK because these guidelines are good enough for now, and definitely intended for all of the UK (including Wales), and we don't need to cut out anything that NHS England might have added that's specific to England.
+
+* Business specifications: We are kickstarting with Apple Human Interface Guidelines: because these orient more toward high-level principles (especially usability and accessibility), rather than low-level rendering (such as colors, fonts, grids).
+
+* Component framework: We are kickstarting shadcn because it is free open source, widespread worldwide, has implementations in JavaScript and TypeScript, and has multiple tunings for specific development frameworks, which in turn improves maintainability and testability.
+
+Ways of working:
+
+* We start now, and we do continuous improvement as we learn more about the real work.
+
+* We understand that all of these options are significantly ahead of NHS Wales DHCW can design and develop internally, because we are resourced constrained.
+
+* We believe that any combination of the above options will be better than none, so we're using the principles of 1) bias for action, 2) don't let perfect be the enemy of the good, 3) ship soon then improve, 4) helping the emergency department staff and getting their direct feedback is a higher priority than any specific pre-planning of colors, fonts, logos, layouts, etc.
```

Comment:
Let's start extracting these principles into a new record

Reply:
Yes good idea. I'll create a new repo for decision records that outside of architecture because IMHO it's important & valuable to have more people in our org participate who are beyond architecture & devops. For example we just started a new data person and I'd like to bias for action on anonymized data here.

Reply:
My preference here would be to leverage this existing repository (and consider the future removal 'architecture' from the name sooner rather than later) - we also have a set of principles coming from our TDA that we should start with ~ so conscious of not presenting the above as 'agreed' yet

Reply:
Suggest we remove the new repos and I can work on a PR for publishing the principles as discussed with our new navigation header/approach